package.json

```diff
@@ -45,24 +45,24 @@
     "@angular/platform-browser": "7.2.15",
     "@angular/platform-browser-dynamic": "7.2.15",
     "@angular/router": "7.2.15",
-    "@blackbaud/auth-client": "2.17.2",
-    "@pact-foundation/pact": "7.4.0",
-    "@pact-foundation/pact-web": "7.4.0",
-    "@skyux-sdk/builder": "3.10.0",
-    "@skyux-sdk/builder-plugin-skyux": "1.0.0",
+    "@blackbaud/auth-client": "2.19.0",
+    "@pact-foundation/pact": "9.5.0",
+    "@pact-foundation/pact-web": "9.5.0",
+    "@skyux-sdk/builder": "3.11.0",
+    "@skyux-sdk/builder-plugin-skyux": "1.2.1",
     "@skyux-sdk/e2e": "3.1.2",
     "@skyux-sdk/pact": "3.3.1",
     "@skyux-sdk/testing": "3.1.0",
     "@skyux/assets": "3.1.0",
     "@skyux/avatar": "3.0.0",
-    "@skyux/config": "3.6.0",
+    "@skyux/config": "3.7.0",
     "@skyux/core": "3.8.0",
     "@skyux/errors": "3.0.0",
-    "@skyux/forms": "3.4.1",
+    "@skyux/forms": "3.4.2",
     "@skyux/http": "3.8.0",
     "@skyux/i18n": "3.9.0",
     "@skyux/indicators": "3.1.0",
-    "@skyux/layout": "3.2.2",
+    "@skyux/layout": "^3.0.0",
```

Comment:
Why are we using a caret here? That goes against what we tell others to do and what we do for all other dependencies.